### ISSUE TITLE
Basic doubt in exercise chapter 10

### ISSUE THREAD

**Issue by ghost** [PacktPublishing/Deep-Reinforcement-Learning-with-Python#3]
Dear all,

I am new to RL and tensorflow and, while replicating the exercise of the Cart Pole Balance with Policy Methods, a doubt arouse regarding the specification of layer 2.
The code says:
```
layer2 = tf.layers.dense(layer1, units=num_actions)
prob_dist = tf.nn.softmax(layer2)
```
I wonder why the softmax function is not passed as 3rd argument to the tf.layers.dense() command.

Apologies if this query is too basic, but recieving some feedback would be highly appreciated.
